orange/src/main/java/com/hartwig/hmftools/orange/algo/util/GermlineConversion.java

```diff
@@ -58,18 +65,30 @@ public static OrangeRecord convertGermlineToSomatic(@NotNull OrangeRecord report
 
     @NotNull
     @VisibleForTesting
-    static PurpleRecord convertPurpleGermline(boolean containsTumorCells, @NotNull PurpleRecord purple) {
+    static PurpleRecord convertPurpleGermline(boolean containsTumorCells, @NotNull PurpleRecord purple)
+    {
         List<PurpleDriver> mergedDrivers;
         List<PurpleVariant> additionalReportableVariants;
-        List<PurpleGainLoss> additionalReportableGainsLosses;
-        if (containsTumorCells) {
-            mergedDrivers = mergeGermlineDriversIntoSomatic(purple.somaticDrivers(), purple.germlineDrivers());
+        List<PurpleGainLoss> mergedAllSomaticGainsLosses;
+        List<PurpleGainLoss> mergedReportableSomaticGainsLosses;
+        List<PurpleGeneCopyNumber> mergedSuspectGeneCopyNumbersWithLOH;
+        if(containsTumorCells)
+        {
+            mergedDrivers =
+                    mergeGermlineDriversIntoSomatic(purple.somaticDrivers(), purple.germlineDrivers(), purple.reportableGermlineFullLosses());
             additionalReportableVariants = toSomaticVariants(purple.reportableGermlineVariants());
-            additionalReportableGainsLosses = toSomaticGainsLosses(purple.reportableGermlineFullLosses());
-        } else {
+            mergedAllSomaticGainsLosses = mergeGermlineFullLosses(purple.reportableGermlineFullLosses(), purple.allSomaticGainsLosses());
```

Comment:
I did it because for the small variants (SAGE) we also only convert the reportable ones. 

We could change this, but then we probably should do the same for small variants and LOH calls. I think this would bloat the JSON a bit without much benefit, but I'm definitely willing to change it if you think it would be better.

Reply:
We initially converted all germline variants but this bloated too much (~3 million variants added). The most challenging of this step was that many variants affect reportable genes and are missense (but very very likely benign). We ended up with many missense variants on the report (as potentially relevant).

For germline losses I can imagine this will not hold. `allSomaticGainsLosses` already contains an entry for every gene in the genome that is either amplified or lost, so this is occasionally a long list already. I don't know exactly how much we would add when adding all germline full losses, but I imagine this will be manageable (certainly less than 3 million :) ). 

I don't have a strong opinion, in practice it matters little I think. Up to you.

Reply:
Ah, makes sense. I'll keep it as is for consistency (and also so I don't have to change it ;p). Thanks!